_data/landscape/meshes.yml

```diff
@@ -123,6 +138,8 @@
   encryption: "Yes"
   multi-tenant-score: 5
   netdev-persona-score: 1
+  spiffe: "?"
```

Comment:
Istio has partial SPIFFE support, but doesn't support the Workload API or Federation API currently, and thus is not interoperable with other SPIFFE implementations.